lib/manageArtifacts.js

```diff
@@ -67,6 +67,7 @@ const defineVariableOperation = (variable) => {
         }
     }
     catch (error) {
+        // Should be throw in case of a wrong format or an unknow operation to proceed
```

Comment:
An error will be thrown when the input doesn't have the expected format, or when the operation is unknown

Reply:
Also, please explain the different edge cases above (if statements) in comments
They're not easy to understand by just looking at the code.

Reply:
To be honest, I don't know myself what could be thrown. Definitely a try/catch to avoid bad operations later in the process.

Reply:
Please change to `An error will be thrown when the input doesn't have the expected format, or when the operation is unknown`